algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,20 @@
+function bfs(value, tree) {
+  let queue = [];
+
+  queue.unshift(tree);
+
+  while (queue.length > 0) {
+    let curNode = queue.pop();
+    if (curNode.value === value) {
+      return curNode;
+    }
+
+    const len = curNode.children.length;
+
+    for (let i = 0; i < len; i++) {
+      queue.unshift(curNode.children[i]);
+    }
+  }
+
+  return null;
+}
```

Comment:
Hi @vale-c 
Can you add an example by passing parameters to the bfs function?
It will be easy for someone to understand the code.

Reply:
@vale-c please work on this asap.

Reply:
Hi @RavinduSachintha ! Sorry but I still haven't found the time to properly work on this issue. I will close this PR and you can assign it to somebody else.